admin/pre_reg/serializers.py

```diff
@@ -10,8 +11,14 @@ def serialize_user(user):
         'id': user._id
     }
 
+
 # TODO: Write and use APIv2 serializer for this
 def serialize_draft_registration(draft, json_safe=True):
+    if draft.approval.meta['registration_choice'] == 'embargo':
+        time = parser.parse(draft.approval.meta['embargo_end_date'])
+        embargo = iso8601format(time) if json_safe else time
+    else:
+        embargo = 'Immediate'
```

Comment:
Not quite sure where I would get this constant from. Am I to construct said constant? @sloria